ui/pages/extensions.js

```diff
@@ -228,6 +228,62 @@ const MesheryDesignEmbedLogo = () => {
   );
 };
 
+const MesheryHelmKanvasLogo = () => {
+  return (
+    <img
+      style={{
+        paddingRight: '1rem',
+        height: 'auto',
+        width: 'auto',
+        maxWidth: '120px',
+        maxHeight: '75px',
+      }}
+      src="/static/img/helm_chart.svg"
+    />
+  );
+};
+
+const MesheryHelmKanvasExtension = () => {
+  const handleLearnMore = (e) => {
+    window.open(
+      'https://artifacthub.io/packages/helm-plugin/helm-kanvas-snapshot/helm-kanvas-snapshot',
```

Comment:
Hmm. Can we point to something within our control, so we don't get broken links that we are unaware of?

In priority order:

1.  a link to the Meshery Docs page for this extension.
2. if that doesn't work (which it should), a link to the repo.
3. if that doesn't work, a link to the meshery.io/extensions page.